openid-4-verifiable-presentations-1_0.md

```diff
@@ -1817,13 +1827,18 @@ precautions against leaking information about the claim value when processing
 In both cases listed here, it needs to be considered that returning an error
 response can also leak information about the processing outcome of `values`.
 
-# Privacy Considerations
+## User Consent and Choice {#user_consent_and_choice}
 
-Many privacy considerations are specific to the credential format and associated proof type used in any particular presentation.
-This section focuses on privacy considerations that are specific to the presentation protocol with some treatment also
-given to considerations that apply across some common credential formats.
+* Wallets should obtain explicit, informed consent from the End-User before releasing any Verifiable Credential or Presentation to a Verifier.
```

Comment:
```suggestion
* Wallets SHOULD obtain explicit, informed consent from the End-User before releasing any Verifiable Credential or Presentation to a Verifier.
```

Reply:
Curious if "must" is a stronger ask and applicable here?